src/traces/scatter/index.js

```diff
@@ -855,3 +857,53 @@ scatter.hoverPoints = function(pointData, xval, yval, hovermode) {
 
     return [pointData];
 };
+
+var DESELECTDIM = 0.2;
+
+scatter.selectPoints = function(searchInfo, polygon) {
```

Comment:
@alexcjohnson could you put this in a separate file? Might as well start to break up `scatter/index.js` as we go.